### PR TITLE
log to stats when items are dropped due to low proba

### PR DESCRIPTION
**Pull request by BurnzZ**
Need this to easily check experiment runs and compare how this stat changes.

### CODE REVIEW

**Review by Gallaecio — APPROVED** [review]

File: zyte_spider_templates/spiders/ecommerce.py [hunk 1]
```diff
@@ -171,6 +171,7 @@ def parse_product(
         if probability is None or probability >= 0.1:
             yield product
         else:
+            self.crawler.stats.inc_value("drop_item/product/low_probability")
```
Comment (PyExplorer) [inline zyte_spider_templates/spiders/ecommerce.py:174]: Just wondering if we have any agreement for stats naming at this time? Should it start from product, for example, like product/drop_item/low_probability for easy tracking in the future when we have several stats for product?

Comment (Gallaecio) [reply zyte_spider_templates/spiders/ecommerce.py:174]: No strong opinion, although I think the proposed naming makes sense. I do wonder if we should have some more prefix (e.g. `spider/ecommerce/`), but no strong opinion on that either.

Comment (BurnzZ) [reply zyte_spider_templates/spiders/ecommerce.py:174]: No strong opinion in the naming for now as patterns will emerge later on.

Comment (PyExplorer) [reply zyte_spider_templates/spiders/ecommerce.py:174]: yes, agree, thanks, let's back to this later on.

**Review by PyExplorer — APPROVED** [review]

**Pull request merged**